src/vs/workbench/contrib/search/browser/searchActionsRemoveReplace.ts

```diff
@@ -375,10 +376,17 @@ export async function getElementToFocusAfterRemoved(viewer: WorkbenchCompressibl
 		while (!!navigator.next() && (!isSearchTreeFolderMatch(navigator.current()) || arrayContainsElementOrParent(navigator.current(), elementsToRemove))) { }
 	} else if (isSearchTreeFileMatch(element)) {
 		while (!!navigator.next() && (!isSearchTreeFileMatch(navigator.current()) || arrayContainsElementOrParent(navigator.current(), elementsToRemove))) {
-			await viewer.expand(navigator.current());
+			// Never expand AI search results by default
+			if (navigator.current() instanceof AITextSearchHeadingImpl) {
+				return navigator.current();
```

Comment:
Is the await viewer.expand line deletion intentional?

Reply:
@rzhao271 ahhh good catch! That was unintentional, reverted it